delivery/src/main/resources/application.yml

```diff
@@ -0,0 +1,18 @@
+server:
+  port: 50054
+
+metadata:
+  application: beachshirts
+  cluster: us-west
+  service: delivery
+  shard: primary
+  customTags:
+    location: Palo-Alto
+    env: production
+
+wavefrontReporting:
```

Comment:
this should not be in no-instrumentation branch.
see https://github.com/wavefrontHQ/sample-app-java/blob/master/shopping/app.yaml